catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/FileSystemPersistenceProvider.java

```diff
@@ -95,7 +96,8 @@ String getPersistencePath() {
      * @return
      */
     String getMapStorePath() {
-        return getPersistencePath() + mapName + File.separator;
```

Comment:
Shouldn't the `getPersistencePath()` method in this implementation also be using `Paths` concatenation instead of String concatenation?

Is there a reason it can't use the `AbsolutePathResolver` as the other implementations do?